Userland/Libraries/LibWeb/Loader/FrameLoader.cpp

```diff
@@ -155,6 +161,11 @@ bool FrameLoader::load(const LoadRequest& request, Type type)
     if (type == Type::IFrame)
         return true;
 
+    auto set_default_favicon = [this] {
+        if (auto* page = browsing_context().page())
+            page->client().page_did_change_favicon(*s_default_favicon_bitmap);
+    };
```

Comment:
Seems like this should be a member function on `FrameLoader` instead of a lambda :)

Reply:
Fixed! :^)